imod_coupler/drivers/ribametamod/config.py

```diff
@@ -19,14 +19,15 @@ class Coupling(BaseModel):
     mf6_active_drainage_packages: Dict[str, str]
     mf6_passive_river_packages: Dict[str, str]
     mf6_passive_drainage_packages: Dict[str, str]
+    mf6_msw_mappings: Dict[str, str] | None = None
 
-    enable_sprinkling: bool  # true whemn sprinkling is active
-    mf6_msw_recharge_pkg: str  # the recharge package that will be used for coupling
-    mf6_msw_well_pkg: (
-        str | None
-    ) = None  # the well package that will be used for coupling when sprinkling is active
-    mf6_msw_node_map: FilePath  # the path to the node map file
-    mf6_msw_recharge_map: FilePath  # the path to the recharge map file
+    enable_sprinkling: Optional[bool] = False  # true whemn sprinkling is active
```

Comment:
```suggestion
    enable_sprinkling: bool = False  # true when sprinkling is active
```
No need for Optional here